### PR TITLE
SUS-4814 | Remove Monobook specific code from Wall

### PR DESCRIPTION
**Pull request by mszabo-wikia**
* notifications handling centralized in `DesignSystemGlobalNavigationWallNotificationsService`
* removed Monobook styles and code paths
* removed dead notifications code

https://wikia-inc.atlassian.net/browse/SUS-4814

### CODE REVIEW

File: extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationWallNotificationsService.class.php [hunk 1]
```diff
@@ -1,5 +1,221 @@
...
+		$this->response->setVal( 'title', $title );
+		$this->response->setVal( 'iso_timestamp', wfTimestamp( TS_ISO_8601, $data->timestamp ) );
+
+		$this->response->setVal( 'isUnread', $unread ? 'unread' : 'read' );
+		if ( $unread && $data->notifyeveryone ) {
+			$this->overrideTemplate( 'NotifyEveryone' );
+		}
+	}
+
+	public function NotificationAdmin() {
```
Comment (macbre) [inline extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationWallNotificationsService.class.php:91]: camelCase? :)

File: extensions/wikia/Wall/Wall.setup.php [hunk 2]
```diff
@@ -196,7 +193,6 @@
 	'wall-button-done-source',
 	'wall-action-*',
 	'wall-message-source',
-	'wall-confirm-monobook-*',
```
Comment (macbre) [inline extensions/wikia/Wall/Wall.setup.php:199]: Should we remove these i18n strings?

**Review by macbre — APPROVED** [review]

File: extensions/wikia/Wall/Wall.setup.php (hunk 2 above)
Comment (mszabo-wikia) [reply extensions/wikia/Wall/Wall.setup.php:199]: they've been removed in this commit 😉

Comment (macbre) [reply extensions/wikia/Wall/Wall.setup.php:199]: Right, GitHub wrapped i18n file diff...

File: extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationWallNotificationsService.class.php (hunk 1 above)
Comment (mszabo-wikia) [reply extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationWallNotificationsService.class.php:91]: I'd leave it be for now as Nirvana's capitalization handling should be addressed separately :D

**Pull request merged**